res/values-vi/strings.xml

```diff
@@ -1028,7 +1051,36 @@
     <string name="ADVANCED_SETUP">CÀI ĐẶT NÂNG CAO</string>
     <string name="WORLD_CUP_EVENT___">Sự kiện WORLD CUP!!!</string>
     <string name="world_cup_event_description">Nhân đôi XP và Plasma trong chế độ bóng đá đến khi hết world cup!</string>
-    <string name="GIF_Skins">GIF Skins</string>
+    <string name="GIF_Skins">Skin GIF</string>
+    <string name="upload_details" translatable="true">
+        "CUSTOM SKINS
+
+When you attempt on uploading a custom skin, the image quality may decrease, but it doesn't matter too much and it is common (you will not notice too much difference while you play). Check a list of status below and their meanings:
+
+In Review (highlighted in yellow): It means your skin is being reviewed by a moderator. You must wait for it to be approved.
+Approved (highlighted in green): It means your skin has been approved. Other players will be able to see your approved skin.
+Refunded (highlighted in orange): It means you got your plasma back because the skin does not meet with the established guidelines of custom skins. You should not try to upload the same again if it happens.
+Rejected (highlighted in red): It means your skin has been rejected and your plasma has been lost, unlike the refunded status. It cannot be seen by third-party.
+
+WHY DID MY SKIN GET REFUNDED OR REJECTED?
+
+As it says on &#x201c;Refunded&#x201d; status, you should not attempt on uploading the same again, otherwise you may have a risk of losing your plasma.
+
+This is a list of content that are *PROHIBITED* to contain on your skin:
+- Skins that contain sexual references;
+- Already existing Nebulous skins, such as level, achievement, season, plasma or exclusive skins from a pack;
+- Skins that contain gore/suicidal references;
+- Skins that contain nudes;
+- Skins that contain someone shirtless;
+- Very dark skins;
+- Totally transparent skins (unidentifiable);
+- Skins that contain depictions of drugs, such as alcohol, cigarette or weed;
+- Skins that contain NSFW pages;
+- Skins that contain necklines;
+- Skins that contain racism, nazism or violence references;
+- Skins that contain usage of dangerous weapons;
+- Skins that contain obscene signs with a hand (it may vary between regions, so be highly careful with it)."
+    </string>
```

Comment:
You might be wondering why I did not base off 100% on the `res/values (English file) for some strings. Some of the newly added strings will be like that in English for higher clarity and that's how people will better understand + is more suitable for translation. I will be marking here the strings that are **not** based 100% on the English file, since inclusions to synonyms that have been used and won't affect meaning/reading they may sound like.